lib/modules/smart_scheduling_links/smart_scheduling_links_basic_sequence.rb

```diff
@@ -488,6 +488,14 @@ def get_file(file, use_token: true)
           assert_valid_http_uri file['url']
         end
 
+        begin
+          Date.iso8601(@manifest['transactionTime'])
+          has_timezone = /(Z|(\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00))/.match(@manifest['transactionTime']).present?
+          assert has_timezone, "Manifest transactionTime '#{has_timezone}' must contain a timezone offset or end in 'Z' for UTC."
+        rescue RuntimeError, ArgumentError
```

Comment:
`TypeError` and `DateError` are the two types of errors I was able to get here. They both inherit from `StandardError`, so you could just rescue that.

```
> Date.iso8601('abc')
Date::Error (invalid date)
> Date.iso8601(123)
TypeError (no implicit conversion of Integer into String)
> Date::Error.ancestors
=> [Date::Error, ArgumentError, StandardError, Exception, Object, Kernel, BasicObject]
> TypeError.ancestors
=> [TypeError, StandardError, Exception, Object, Kernel, BasicObject]
```